### DISCUSSION TITLE
Detectron2TestConstants missing faster_rcnn_R_50_FPN_3x.yaml config

### DISCUSSION THREAD

**Discussion by Akhp888** [obss/sahi#630] [Q&A] [answered]
Hi,

I am testing sahi for detectron and my model was trained on faster_rcnn_R_50_FPN_3x.yaml from detectron zoo , now i want to infer using sahi sliced prediction but unfortunately i dont find the config available in Detectron2TestConstants module.

Thanks

**Comment by fcakyon** [comment]
Thanks for your interest!

You have to provide config path from your local. Test constants are for demo and continious integration purposes only 👍

**Comment by Akhp888** [comment]
Thanks for the reply , 
will test that :)

**Comment by fcakyon** [comment, marked as answer]
@Akhp888 check detectron2+SAHI blogpost on how to provide custom config files :)
https://medium.com/codable/perform-sliced-inference-and-detailed-error-analysis-using-detectron2-models-37063e33073f

**Comment by Akhp888** [comment]
@fcakyon Thanks ,

I want to know if there is any ground rule for setting the slice_height and slice_width based on the train time img_size ?
As i find a lot of comparably (w.r.t other classes) smaller objects dont get detected.
